pyaedt/hfss.py

```diff
@@ -4600,22 +4140,22 @@ def assign_radiation_boundary_to_objects(self, obj_names, boundary_name=""):
 
         """
 
-        object_list = self.modeler.convert_to_selections(obj_names, return_list=True)
-        if boundary_name:
-            rad_name = boundary_name
+        object_list = self.modeler.convert_to_selections(assignment, return_list=True)
+        if name:
+            rad_name = name
         else:
             rad_name = generate_unique_name("Rad_")
         return self.create_boundary(self.BoundaryType.Radiation, object_list, rad_name)
 
-    @pyaedt_function_handler()
-    def assign_hybrid_region(self, obj_names, boundary_name="", hybrid_region="SBR+"):
+    @pyaedt_function_handler(obj_names="assignment", boundary_name="name")
+    def assign_hybrid_region(self, assignment, name="", hybrid_region="SBR+"):
         """Assign a hybrid region to one or more objects.
 
         Parameters
         ----------
-        obj_names : str or list or int or :class:`pyaedt.modeler.cad.object3d.Object3d`
+        assignment : str or list or int or :class:`pyaedt.modeler.cad.object3d.Object3d`
             One or more object names or IDs.
-        boundary_name : str, optional
+        name : str, optional
             Name of the boundary. The default is ``""``.
         hybrid_region : str, optional
             Hybrid region to assign. Options are ``"SBR+"``, ``"IE"``, ``"PO"``. The default is `"SBR+"``.
```

Comment:
```suggestion
            Hybrid region to assign. The default is `"SBR+"``. Options are ``"IE"``, ``"PO"``
            and ``"SBR+"``. 
```
Default first--and then list of options in alphabetical order.